### PR TITLE
Manage organisations

### PR DESCRIPTION
**Pull request by tombu**

### CODE REVIEW

**Comment by tombu** [comment]
When testing the API with the frontend there was an issue like described here: https://github.com/cerebris/jsonapi-resources/issues/670

A proposed solution for this is to fake it.
```
def contact_person_id=(id)
  # this helper fixes a bug in jsonapi-resources 0.7.1.beta1
  # https://github.com/cerebris/jsonapi-resources/issues/670
end
````

Maybe there is a solution to fix it in the frontend – I just didn´t find one.

File: app/adapters/application.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import ENV from '../config/environment';
+import DS from "ember-data";
+// import Ember from "ember";
```
Comment (stravid) [inline app/adapters/application.js]: What's with all the commented out stuff?

File: app/routes/v2/clients/edit.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  model(parameters) {
+    return this.store.findRecord('organisation', parameters.id);
+  },
+
+  actions: {
+    saveOrganisation() {
+      const model = this.modelFor('v2.clients.edit');
+
+      model.save().then(() => {
+        return model.get('contactPerson.content').save();
```
Comment (stravid) [inline app/routes/v2/clients/edit.js]: I don't think it's a best practice to access `content` of a promise. Actually I think it's a bad practice.

Does `model.get("contactPerson").then(contactPerson => { return contactPerson.save(); }).then ...` work?

If it does, can we change all occurrences of this pattern?

Comment (tombu) [inline app/routes/v2/clients/edit.js]: Thanks for your help on that. I didn´t remember how to do it correctly :( Accessing `content` was just a lack of knowledge.

**Pull request merged**